### PR TITLE
Keep existing bower resolutions for P3 conversion

### PR DESCRIPTION
**Pull request by web-padawan**
This change is [Reviewable](https://reviewable.io/reviews/vaadin/magi-cli/41)

### CODE REVIEW

File: bin/magi-p3-convert [hunk 1]
```diff
@@ -72,7 +72,8 @@ async function main() {
     // TODO(web-padawan): revert commit once the next 2.x release after 2.6.0 lands
     const pinPolymer = '5f5d2c2';
     bowerJson.dependencies.polymer = `Polymer/polymer#${pinPolymer}`;
-    bowerJson.resolutions = {polymer: `${pinPolymer}`};
+    const resolutions = bowerJson.resolutions || {};
+    bowerJson.resolutions = {polymer: `${pinPolymer}`, ...resolutions};
```
Comment (limonte) [inline bin/magi-p3-convert]: should the order be opposite? `{...resolutions, polymer: ${pinPolymer}}`

image

Comment (web-padawan) [reply bin/magi-p3-convert]: Done.

File: bin/magi-p3-convert [hunk 2]
```diff
@@ -72,7 +72,8 @@ async function main() {
     // TODO(web-padawan): revert commit once the next 2.x release after 2.6.0 lands
```
Comment (limonte) [inline bin/magi-p3-convert]: BTW, maybe not directly related to this PR, but 2.6.0 landed https://github.com/Polymer/polymer/releases/tag/v2.6.0

Can we maybe remove this workaround at all?

Comment (web-padawan) [reply bin/magi-p3-convert]: "the next release after 2.6.0" is still not tagged

Comment (limonte) [reply bin/magi-p3-convert]: oops, misread, sorry.

**Pull request merged**